### PR TITLE
Add picking_info parameter to documentation of update stock picking A…

### PR DESCRIPTION
**Pull request by CallumCoombes**
…PI endpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ Update/mutate the stock picking
 * @param (optional) result_package_name - If it corresponds to an existing package/pallet that is not in an other location, we will set it to the `result_package_id` of the operations of the picking (i.e. transfer). If the target storage format of the picking type is pallet of packages it will set `result_parent_package_id`.
 * @param (optional) package_name - Name of the package of the picking to be marked as done
 * @param (optional) products_info - An array with the products information to be marked as done, where each dictionary contains: product_barcode, qty and serial numbers if needed
-
+* @param (optional) picking_info - A dictionary of generic picking information to update the stock picking with
```
Comment (ws-mps) [inline README.md]: Have you double checked if this parameter name is correct or if it has to be called "transport" and only set the transport related information?

Comment (CallumCoombes) [reply README.md]: After discussion it was decided to leave it as picking_info and add the trailer information to it.

**Review by ws-mps — APPROVED** [review]

File: models/stock_picking.py [hunk 2]
```diff
@@ -61,20 +61,28 @@ def is_compatible_package(self, package_name):
 
         return res
 
-    def update_picking(self, expected_package_name=None, **kwargs):
+    def update_picking(self, expected_package_name=None, u_transport_id=None, **kwargs):
```
Comment (readikus) [inline models/stock_picking.py]: Why isn't the transport info in kwargs?

File: models/stock_picking.py [hunk 3]
```diff
@@ -61,20 +61,28 @@ def is_compatible_package(self, package_name):
...
         """ Extend update_picking with a new parameter expected_package_name
             to be used during swapping packages
             TODO: finish implement and add parameter at README
         """
+        # if picking_info already exists in kwargs, update with trailer info
```
Comment (readikus) [inline models/stock_picking.py]: Should be in the kwargs,not a separate param

File: models/stock_picking.py (hunk 2 above)
Comment (readikus) [inline models/stock_picking.py]: Also, why is expected_package_name not in kwargs?

**Review by readikus — CHANGES_REQUESTED** [review]

File: models/stock_picking.py (hunk 3 above)
Comment (CallumCoombes) [reply models/stock_picking.py]: moved to kwargs

File: models/stock_picking.py (hunk 2 above)
Comment (CallumCoombes) [reply models/stock_picking.py]: This is being handled in branch udes-story/528

Comment (CallumCoombes) [reply models/stock_picking.py]: moved to kwargs

File: models/stock_picking.py (hunk 3 above)
Comment (CallumCoombes) [reply models/stock_picking.py]: This is being handled in branch udes-story/528

**Pull request merged**